tests/test_query.py

```diff
@@ -206,16 +196,14 @@ def test_big_and_small_full_separate_small_files(self):
 
         # if a uuid has related files or annotations then they
         # are downloaded as big files
-        assert bigs == ['big', 'small']
+        assert set(bigs) == set(['big', 'small'])
```

Comment:
don't typecast here

Reply:
The reason for this is, that the order might be messed up, even though the `bigs` might be `bigs == ['small', 'big']` this assertion won't work. Converting it to set, just makes sure that we don't need to worry about the order. Or do we?